### ISSUE TITLE
Here's detailed data from Italy

### ISSUE THREAD

**Issue by epignatelli** [CSSEGISandData/COVID-19#484]
Thanks guys for this tireless work.

I don't know if you are aware, but here's detailed data from the Italian Protezione Civile (Italian's emergency response corpus):
http://opendatadpc.maps.arcgis.com/apps/opsdashboard/index.html#/b0c68bce2cce478eaac82fe38d4138b1

This repo contains the raw data.
https://github.com/pcm-dpc/COVID-19

Do let me know if you need help with translation to English.

**Comment by AnthonyEbert** [comment]
Duplicate of #335

**Comment by rladeira** [comment]
Are there similar repositories, with detailed information, for others Europe countries?

**Comment by JiPiBi** [comment]
@rladeira 

I was searching for such data for France and didnt' find . The Italian work for producing data is very impressive and should be a model to be followed by all other countries for their official reporting, and so allowing easy aggregation for WHO or others

**Comment by rladeira** [comment]
@JiPiBi, i agree with you. Italy has been doing an amazing job submitting detailed data in a machine readable way.

**Comment by JiPiBi** [comment]
@rladeira 
Correction :  https://www.santepubliquefrance.fr/recherche/#search=COVID-19%20:%20point%20epidemiologique can give information by region and also you can find information about the stock of people into reanimation but globally , BUT
- informations are not in csv or data form , only in pdf (perhaps some tools could extract data from pdf ??)
- data are no more daily updated but weekly (following their answer to my question)  and with the current pace could be a bit slow..

**Comment by giocaputo** [comment]
here there are italian repository, day by day,  of Dipartimento della Protezione Civile in csv and json format.
https://github.com/pcm-dpc/COVID-19

**Comment by pabramowitsch** [comment]
Does anyone know if the code behind the opendatadpc UI for Italy is also available?  Or know the platform?  Is it in RShiny?
